io/disk/vfc-tests/virtual_fc.py

```diff
@@ -55,6 +55,7 @@ def setUp(self):
         self.hmc_pwd = self.params.get("hmc_pwd", '*', default=None)
         self.hmc_username = self.params.get("hmc_username", '*', default=None)
         self.count = self.params.get("count", default=1)
+        self.skip_host = self.params.get("skip_host", default=None)
```

Comment:
@Naresh-ibm Is there a possibility to find the bootable host through sysfs or commands instead of getting this as an input ?

Reply:
Probably we can, but there are some scenarios where this fails.
vfc testing is based adapter, I mean we need to test only those vfchost  which are configured from a desired adapter. where the bootable disks are not required to be tested. In some scenarios OS will be on non-vfc disk (even not on bootable disk) at that time our logic of checking its sysfs or OS install disk will fail.

So the easiest way to handle this is just skipping and run the script on specific vfchosts.

Reply:
@Naresh-ibm What I meant to ask was, if there is a way bootable host can be found without user input, we can skip that host irrespective of whether that host is from our test adapter or not.

Reply:
No way. we must need atleast one parameter from yaml for sure for sort out , because we have already removed many other parameters from yaml. So we need the input from yaml for sure.

Reply:
Okay, this should be done in the future, if not now. Can you please mark as TODO ?

Reply:
@Naresh-ibm is this marked as TODO ?